_projects/lomap.md

```diff
@@ -0,0 +1,11 @@
+---
+name: lomap
+repo: OpenFreeEnergy/Lomap
+conda-install: lomap2
+role: maintainers
+description: >-
+  Lomap is a tool for creating atom mappings between two ligand molecules.
+  It was originally developed by the research lab of David Mobley at
```

Comment:
Can we have a link to the original paper and maybe a link to the defunct repo?

Reply:
For the defunct repo, adding a link in the Lomap repo README in https://github.com/OpenFreeEnergy/Lomap/pull/12 -- I don't think we need that here as well. I will add a link to the paper here; checking that I have the right reference (see  also https://github.com/OpenFreeEnergy/Lomap/pull/11).

Reply:
link to paper added